libs/pilight/protocols/433.92/alecto_ws1700.c

```diff
@@ -75,13 +75,17 @@ static void parseCode(void) {
 		}
 	}
 
-	id = binToDecRev(binary, 0, 11);
+	header = binToDecRev(binary, 0, 3);
+	if (header != 5) {
+		return;
+	}
+	id = binToDecRev(binary, 4, 11);
 	battery = binary[12];
-	temperature = ((double)binToDecRev(binary, 18, 27));
+	temperature = (double)binToDecRev(binary, 16, 27);
 	humidity = (double)binToDecRev(binary, 28, 35);
 
-	if(temperature > 511) {
-		temperature -= 1023;
+	if (binary[16]) {
+	    temperature = temperature - 4096;
```

Comment:
Pffff…

Have to read that over (and over again..).

I have cloned pilight with GitHub to my own repository on my local machine.

From that I have cloned (from my local machine) the develop branche with “git clone" to my RPi.
There I have made the changes to the alecto_ws1700.c file, committed it with “git commit -a” and then pushed the changes to my local machine.
With GitHub I than reguested a “push-request”…

As I said, quit new to this Git(Hub) thingy...

I will give it a try in a bit.

Regards,

Willem

gsm: 0653 - 499 551
http: www.aandewiel.nl

---
:gobble: /vt./  1. To consume, usu. used with `up'.  "The
   output spy gobbles characters out of a {tty} output buffer."
   2. To obtain, usu. used with `down'.  "I guess I'll gobble down
   a copy of the documentation tomorrow."  See also {snarf}.

> On 15 jun. 2016, at 13:44, pilino1234 <notifications@github.com> wrote:
> 
> In libs/pilight/protocols/433.92/alecto_ws1700.c <https://github.com/pilight/pilight/pull/287#discussion_r67146322>:
> 
> >  	humidity = (double)binToDecRev(binary, 28, 35);
> >  
> > -	if(temperature > 511) {
> > -		temperature -= 1023;
> > +	if (binary[16]) {
> > +	    temperature = temperature - 4096;
> Are you using Github to edit the code, or are you using git?
> 
> If you're using git you can make a new commit, and the follow these instructions to "squash" this current commit with the new one for the -= operator:
> http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit <http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit>
> Then you run
> git push origin HEAD -f
> to force push your change to the same branch. GitHub will automatically update the pull request.
> 
> Whatever you do, just please don't close this PR and open a new one 😉
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/pilight/pilight/pull/287/files/f74c09ce59a0ffa3dfe12accbe3a1a1bf8580887#r67146322>, or mute the thread <https://github.com/notifications/unsubscribe/AFU6E_m_c6Ud40O7Pt9yrYyTS66B6yM7ks5qL-W1gaJpZM4I2DgO>.
>